test/sharness/t0040-add-and-cat.sh

```diff
@@ -77,6 +77,17 @@ test_add_cat_file() {
         echo "added $HASH hello.txt" >expected &&
         test_cmp expected actual
     '
+
+    test_expect_success "ipfs add on hidden file succeeds" '
+    	echo "Hello Worlds!" >mountdir/.hello.txt &&
+        ipfs add mountdir/.hello.txt >actual
+    '
+
+    test_expect_success "ipfs add on hidden file output looks good" '
+    	HASH="QmVr26fY1tKyspEJBniVhqxQeEjhF78XerGiqWAwraVLQH" &&
+        echo "added $HASH .hello.txt" >expected &&
```

Comment:
The line above also seems to have an indentation issue. Or perhaps it is the line above it?